test/test_distributions.py

```diff
@@ -1,11 +1,20 @@
 from common import TestCase, run_tests
 import math
+import numpy as np
+import scipy.stats
+import scipy.special
```

Comment:
While using `numpy` and `scipy` is fine, their imports should be guarded with a `try` statement, and if it fails because of `ImportError` then you should skip the tests (instead of a hard failure)